Dockerfile

```diff
@@ -44,7 +44,13 @@ RUN --mount=type=cache,target=/go/pkg/mod \
     --mount=type=cache,target=/root/.cache/go-build \
     CGO_ENABLED=0 go build -trimpath -ldflags "-s -w -X 'main.version=${ATLANTIS_VERSION}' -X 'main.commit=${ATLANTIS_COMMIT}' -X 'main.date=${ATLANTIS_DATE}'" -v -o atlantis .
 
-FROM debian:${DEBIAN_TAG} as debian-base
+FROM debian:${DEBIAN_TAG} AS debian-base
+
+# Set up the 'atlantis' user and adjust permissions. User with uid 1000 is for backwards compatibility
+RUN useradd --uid 100 --system --create-home --user-group --shell /bin/bash atlantis && \
+    useradd --uid 1000 --system --home=/home/atlantis/ --groups atlantis --shell /bin/bash atlantis2 && \
```

Comment:
@kvanzuijlen we noticed that new group id for `atlantis` is now `998` instead of keeping`1000`. Can you please create the group before so we can ensure that is kept too? 🙏

Reply:
Sure! I'll take a look to see if it's possible with useradd, if not, I'll use groupadd.

Reply:
Pinned the gid to 1000 for both alpine and debian, same as is configured in the Helm Chart.

Reply:
And thanks for catching this!